docker/scripts/release.sh

```diff
@@ -25,9 +25,9 @@ create_git_tag() {
 }
 
 retag_and_push_image() {
-    IMAGE="${LOKALISE_REGISTRY}/${GIT_REPO_NAME}/${APP_NAME}"
+    IMAGE="${LOKALISE_ECR_REGISTRY}/${GIT_REPO_NAME}/${APP_NAME}"
```

Comment:
instead of modifying the function, better to create a new one like `retag_and_push_ecr`

Reply:
Doing that, thanks